static/app/components/events/autofix/preferences/components/SelectableRepoItem.tsx

```diff
@@ -0,0 +1,117 @@
+import styled from '@emotion/styled';
+
+import {Tooltip} from 'sentry/components/tooltip';
+import {IconAdd} from 'sentry/icons';
+import {t} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import type {Repository} from 'sentry/types/integrations';
+
+interface Props {
+  isSelected: boolean;
+  onToggle: () => void;
+  repo: Repository;
+}
+
+export function SelectableRepoItem({repo, isSelected, onToggle}: Props) {
+  const isGithub = repo.provider?.name?.toLowerCase() === 'github';
+
+  const tooltipMessage = isGithub
+    ? ''
+    : t('Support for %s will be coming soon', repo.provider?.name || t('this provider'));
+
+  const repoContent = (
+    <RepoListItemContainer
+      selected={isSelected}
+      onClick={() => isGithub && onToggle()}
+      disabled={!isGithub}
+      role="button"
+      tabIndex={0}
+      data-repo-id={repo.externalId}
+    >
+      <RepoHeader>
+        <RepoInfoWrapper>
+          <RepoName>{repo.name}</RepoName>
+          <RightAlign>
+            <RepoProvider>{repo.provider?.name || t('Unknown Provider')}</RepoProvider>
+            {isGithub && <AddIcon size="xs" />}
```

Comment:
should the icon disappear, or be grayed out?

Reply:
hmm i think disappear gives a bigger distinction but i don't mind either lol